web/src/app/admin/admin-alert-counts/AlertCountControls.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useMemo } from 'react'
+import {
+  Button,
+  Grid,
+  Select,
+  InputLabel,
+  MenuItem,
+  FormControl,
+  SelectChangeEvent,
+  Card,
+} from '@mui/material'
+import ResetIcon from '@mui/icons-material/Replay'
+import { ISODateTimePicker } from '../../util/ISOPickers'
+import { useURLParams, useResetURLParams } from '../../actions'
+import { DateTime } from 'luxon'
+
+export default function AlertCountControls(): JSX.Element {
+  const now = useMemo(() => DateTime.now(), [])
+  const [params, setParams] = useURLParams({
+    createdAfter: now.minus({ days: 1 }).toISO(),
+    createdBefore: '',
+    interval: 'PT1H',
+  })
+
+  const handleFilterReset = useResetURLParams(
+    'createdAfter',
+    'createdBefore',
+    'interval',
+  )
+
+  return (
+    <Card>
+      <Grid container spacing={1} sx={{ padding: 2 }}>
+        <Grid item>
```

Comment:
could we play around with the grid's item layout a little bit? or maybe min-width? i think it would make more sense to have the date pickers be a bit longer in width, and have the select flex to the remaining from that?

this is with setting the min-width to `325px` for the selects and `150px` for the interval dropdown:

![Screen Shot 2022-07-21 at 8 38 43 AM](https://user-images.githubusercontent.com/11381794/180255088-243f89d3-80a4-4116-a8df-1972648a5130.png)